guides/common/modules/proc_monitoring-server.adoc

```diff
@@ -0,0 +1,15 @@
+[id="monitoring-{project-context}-server_{context}"]
+= Monitoring {ProjectServer}
+
+Audit records list the changes made by all users on {Project}.
+This information can be used for maintenance and troubleshooting.
+
+.Procedure
+* In the  {ProjectWebUI}, navigate to *Monitor* > *Audits* to view the audit records.
+
+To obtain a list of all the audit attributes, use the following command:
++
+[options="nowrap"]
+----
+# foreman-rake audits:list_attributes
```

Comment:
I don't think we should write this twice; maybe delete line 9-end?

Reply:
I'd go even further and say that the entire module should be deleted since it has the almost exact same contents as the other file. You can keep the header and put it into the other file instead.

Reply:
Hey @adamlazik1 I have actually gotten rid of the two separate modules and just kept the one module (which I changed from a con module to proc. Does that seem reasonable or should we have a separate module for audit logging?

Reply:
@AkshayGadhaveRH I you review the `Files changed` section you will find that there are still two files with the same content being added:
`guides/common/modules/proc_monitoring-server.adoc`
and
`guides/common/modules/proc_viewing-audit-logging-information.adoc`
Perhaps you deleted one locally but change change was somehow omitted from the commit?

Reply:
@adamlazik1 that's strange, let me push another commit.